app/views/collections/edit.html.erb

```diff
@@ -89,4 +89,15 @@
 <% end %>
 </ul>
 
-<%= render 'edit_javascript' %>
+<script>
+pdc = {
+  ...pdc,
```

Comment:
What does the `...pdc` syntax do?

Reply:
oh! I see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax